### ISSUE TITLE
docs.webp.sh

### ISSUE THREAD

**Issue by mathieu-aubin** [webp-sh/docs#1]
What is wrong with the website being only partially updated?

That readme is AGES old and even shows a build fail icon ... ??

**Comment by n0vad3v** [comment]
Yes, that's indeed a problem, I've made several changes to the website as it will show more accurate information, see commit: https://github.com/webp-sh/docs/commit/444a7c4272314930a1abf5791ae534a9f2e32099
